content/spin/kv-store-tutorial.md

```diff
@@ -22,27 +20,14 @@ url = "https://github.com/fermyon/developer/blob/main//content/spin/kv-store.md"
-
-> Spin users can now persist and retrieve non-relational data from a key/value store across multiple requests to and from the same application; written in any of these languages.
-
-### Redis, PostgreSQL & SQLite
-
-A while back, around the Spin v0.5.0 version, we published an article on [Persistent Storage in Webassembly Applications](https://www.fermyon.com/blog/persistent-storage-in-webassembly-applications). In that previous article, we showed the Spin framework's capabilities of providing WebAssembly executables with access to different levels of on-disk persistence. We demonstrated manually installing Redis from source, running our Redis server on localhost and then reading and writing from Redis via both the Redis CLI itself and also via the Spin SDK. Our current documentation also has examples of using both [Redis](https://developer.fermyon.com/cloud/data-redis) and [PostgreSQL](https://developer.fermyon.com/cloud/data-postgres), via [Redis Labs](https://redis.com/) and [ElephantSQL](https://www.elephantsql.com/plans.html) services respectively. 
-
-In all of these previous examples, to persist data within your application, a separate data storage layer is a requirement. These methods of data persistence are perfectly fine and sound to use as part of your application, if you choose to do so.
-
-However, from Spin v0.9.0 onwards, you are no longer required to install any on-disk persistence outside of just installing Spin itself. SQLite is embedded inside Spin in a way that is analogous to how one would imagine using a software library. When you run your Spin application using the [spin up](https://developer.fermyon.com/common/cli-reference#up) command, SQLite is available to your application. When your application is no longer running, neither is SQLite. Your data will continue to persist at all times (including during restarts) and will support the running of your application.
-
-In the future, it may be possible to embed other database technologies into Spin. For example, future Spin SDK updates may target other databases such as Redis, which would allow you to build applications using Redis via minimal Spin configuration without the need to locally install and maintain your own Redis instance.
-
-Let's get started with creating and deploying your first Spin application that uses this new key/value storage mechanism.
+* Create a Spin application with `spin new`
+* Use the key value store SDK to get, set, and list key value pairs
+* Configure your application manifest (`spin.toml`) to use the default key value store
+* Run your key value store enlighted Spin application locally with `spin up`
```

Comment:
Please clarify or remove "enlighted" - I am not sure if it is meant to be "enlightened" but if it is then I still don't know what it means in this context, sorry